config/20-bosco.ini

```diff
@@ -0,0 +1,29 @@
+
+
+;===================================================================
+;                              Bosco
+;===================================================================
+
+[BOSCO]
+; This section has settings for configuring your CE for a Slurm job manager
+
+; (Required) Determines whether you want a bosco route for the compute element
+enabled = FALSE
+
+; (Required) A comma separated list of users for which to enable bosco submission
+users = UNAVAILABLE
+
+; (Required) The endpoint should be the hostname of the remote login node
+endpoint = UNAVAILABLE
+
+; (Required) The type of batch system used on the remote cluster.  Possible values are:
+; pbs, lsf, sge, condor
+; If the remote system is Slurm, put pbs as the batch
```

Comment:
I don't like the "If the remote system is Slurm, put pbs as the batch" thing. What if we want to add code to properly handle slurm later? The user shouldn't have to go and change their config in that case.

The user shouldn't have to be aware that bosco treats slurm as pbs. IMO that's an implementation detail.